packages/dx-react-scheduler-material-ui/src/templates/utils.js

```diff
@@ -30,3 +30,15 @@ export const scrollingStrategy = (scrollablePart, fixedPart) => {
 };
 
 export const setColor = (level, color) => (color[level] || PRIMARY_COLOR[level]);
+
+export const getResourceColor = (resources) => {
+  if (resources && resources.length) {
+    return resources.find(resource => resource.isMain).color;
+  } return undefined;
+};
+
+export const getAppointmentColor = (level, color, defaultColor) => {
+  if (!color) return setColor(level, defaultColor);
+  if (typeof color === 'string') return color;
+  return setColor(level, color);
```

Comment:
`setColor` does not actually sets color. May be better `ensureColor` or similar?